packages/rest/lib/rest-decorators.cadl

```diff
@@ -5,6 +5,7 @@ using Cadl.Reflection;
 extern dec autoRoute(target: Namespace | Interface | Operation);
 extern dec segment(target: object | ModelProperty | Operation, name: string);
 extern dec segmentOf(target: Operation, type: object);
+extern dec actionSeparator(target: object | ModelProperty | Operation, seperator: string);
```

Comment:
Should we give a type to `seperator` `"/"| ":"|"/:"` to be the same as the allowed list in ts?

Reply:
Nice. That lets me remove the custom diagnostic as well.